searchapp/templates/home.html

```diff
@@ -21,19 +21,19 @@
     </div>
   <div class="ui grid">
     <div class="four wide column">
-      <div class="button" id="test"><a href="student_view" class="button">Test Paper</a>
+      <div class="button" id="test"><button onclick="set_paper_type('test')">Test Paper</button>
```

Comment:
Create a .js file for home.html and use `.click` function in it to get the worksheet type based on `id`. Remove the outer div with class button. Move id from `<div></div>` to `<button></button>` and have the same class . This will help to have a single function for all 3 and worksheet type can be set based on id.